pytorch_lightning/lite/lite.py

```diff
@@ -78,8 +88,10 @@ def __init__(
         gpus: Optional[Union[List[int], str, int]] = None,
         tpu_cores: Optional[Union[List[int], str, int]] = None,
     ) -> None:
-        self._check_accelerator_support(accelerator)
-        self._check_strategy_support(strategy)
+        self._check_accelerator_flag(accelerator)
```

Comment:
To Reviewers: I couldn't find any other way to validate this, but I think here it is fine to use the types, as this is for input validation and not for checking which mode we are in.